util/customTransforrmationsStore-v1.js

```diff
@@ -5,9 +5,9 @@ const stats = require("./stats");
 const transformationCache = {};
 const libraryCache = {};
 
-const CONFIG_BACKEND_URL =
-  process.env.CONFIG_BACKEND_URL || "https://api.rudderlabs.com";
-
+// const CONFIG_BACKEND_URL =
+//   process.env.CONFIG_BACKEND_URL || "https://api.rudderlabs.com";
+const CONFIG_BACKEND_URL = "http://localhost:5000";
```

Comment:
Can we revert this change ?

Reply:
yes reverted for now. Although it needs to be set to the dev config-be URL for the time being, since its going to be merged to develop first